docs/standard/garbage-collection/latency.md

```diff
@@ -7,42 +7,50 @@ helpviewer_keywords:
+
+During low latency periods, generation 2 collections are suppressed unless the following occurs:
 
 - The system receives a low memory notification from the operating system.
 
-- Your application code induces a collection by calling the <xref:System.GC.Collect%2A?displayProperty=nameWithType> method and specifying 2 for the `generation` parameter.
+- Application code induces a collection by calling the <xref:System.GC.Collect%2A?displayProperty=nameWithType> method and specifying 2 for the `generation` parameter.
 
- The following table lists the application scenarios for using the <xref:System.Runtime.GCLatencyMode> values.
+## Scenarios
+
+The following table lists the application scenarios for using the <xref:System.Runtime.GCLatencyMode> values:
 
 |Latency mode|Application scenarios|
 |------------------|---------------------------|
-|<xref:System.Runtime.GCLatencyMode.Batch>|For applications that have no UI or server-side operations.<br /><br /> This is the default mode when [concurrent garbage collection](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md) is disabled.|
-|<xref:System.Runtime.GCLatencyMode.Interactive>|For most applications that have a UI.<br /><br /> This is the default mode when [concurrent garbage collection](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md) is enabled.|
-|<xref:System.Runtime.GCLatencyMode.LowLatency>|For applications that have short-term, time-sensitive operations during which interruptions from the garbage collector could be disruptive. For example, applications that do animation rendering or data acquisition functions.|
-|<xref:System.Runtime.GCLatencyMode.SustainedLowLatency>|For applications that have time-sensitive operations for a contained but potentially longer duration of time during which interruptions from the garbage collector could be disruptive. For example, applications that need quick response times as market data changes during trading hours.<br /><br /> This mode results in a larger managed heap size than other modes. Because it does not compact the managed heap, higher fragmentation is possible. Ensure that sufficient memory is available.|
+|<xref:System.Runtime.GCLatencyMode.Batch>|For applications that have no user interface (UI) or server-side operations.<br /><br />This is the default mode for both server and workstation garbage collection when background garbage collection is disabled.|
+|<xref:System.Runtime.GCLatencyMode.Interactive>|For most applications that have a UI.<br /><br />This is the default mode for workstation and server garbage collection for a standalone executable that's not hosted.<br /><br />This is the default mode for both server and workstation garbage collection when background garbage collection is enabled.|
```

Comment:
Having the two "This is the default" lines is confusing, especially as they're out of sync in listing server and workstation in different orders. Does the first line now override the second line, i.e., even if background collection isn't enabled it's the default in the line 1 scenario? 

Also, I presume if an app is "hosted," the GC setting of the hosting process takes precedence; if so should we explicitly state that?